SwiftWisdom/Core/StandardLibrary/Array/Array+Utilities.swift

```diff
@@ -54,3 +54,43 @@ extension Array {
         }
     }
 }
+
+extension CollectionType {
+    public var ip_middleValues: [Generator.Element]? {
```

Comment:
Other average functions depend on this one, so we could make it internal or something while we think about it. I'd prefer to figure out a way to have this make sense. This does not use comparability, but I don't know how to say that.